doc/user_interface.md

```diff
@@ -0,0 +1,616 @@
+# User interface
+
+## General principles
+
+The Visual Query Builder interface breaks down into 2 panels:
+
+- the **transformation pipeline on the left**: this is where you can see the
+  ordered series of transformation steps that are applied to your data. When you
```

Comment:
```suggestion
  ordered serie of transformation steps that are applied to your data. When you
```

Reply:
@adimasci serie does not exist. It is always series :)